docs/source/athena-deploy.rst

```diff
@@ -0,0 +1,97 @@
+Athena Deployment
+=================
+
+Overview
+--------
+
+After the initial `Athena Setup <athena-setup.html>`_, a new Lambda function must be deployed.
+
+The name of the Lambda function is Athena Partition Refresh.
+
+The purpose of this function is to periodically refresh Athena tables to enable the search-ability of data.
+
+The default refresh interval is 10 minutes.
+
+Concepts
+--------
+
+This function utilizes:
+
+* `AWS S3 Event Notifications <http://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html>`_
+* `AWS SQS <https://aws.amazon.com/sqs/details/>`_
+* `AWS Lambda Invocations by Schedule <http://docs.aws.amazon.com/lambda/latest/dg/tutorial-scheduled-events-schedule-expressions.html>`_
+* `AWS Athena Repair Table <http://docs.aws.amazon.com/athena/latest/ug/ddl/msck-repair-table.html>`_
+
+Diagram
+~~~~~~~
+
+.. figure:: ../images/athena-refresh-arch.png
+  :alt: StreamAlert Athena Refresh Partition Diagram
+  :align: center
+  :target: _images/athena-refresh-arch.png
+
+Internals
+~~~~~~~~~
+
+Each time the Athena Partition Refresh Lambda function starts up, it does the following:
+
+* Polls the SQS Queue for the latest S3 event notifications (up to 50)
```

Comment:
This document doesn't detail what the s3 event notifications represent/mean. Can you clarify this a bit here or earlier?